scipy/stats/stats.py

```diff
@@ -3486,9 +3485,15 @@ def kendalltau(x, y, initial_lexsort=None, nan_policy='propagate'):
 
     References
     ----------
-    W.R. Knight, "A Computer Method for Calculating Kendall's Tau with
-    Ungrouped Data", Journal of the American Statistical Association, Vol. 61,
-    No. 314, Part 1, pp. 436-439, 1966.
+    .. [1] Maurice G. Kendall, "A New Measure of Rank Correlation", Biometrika
+    Vol. 30, No. 1/2, pp. 81-93, 1938.
```

Comment:
The price for not having to write `\item` is the darn thing being *very* picky about whitespace. I'm guessing `Vol` needs to line up with `Maurice`. Otherwise, 

https://1150-1460385-gh.circle-artifacts.com/0/home/ubuntu/scipy/doc/build/html-scipyorg/generated/scipy.stats.kendalltau.html#scipy.stats.kendalltau